docs/00-prerequisite.md

```diff
@@ -2,11 +2,10 @@
 
 > Intended commands to be executed from the root directory of this repository. The majority of the configurations to be applied are already created, with the exception of the ones that prompts you for specifics that are either created in the command or dumped to a `scratch` dir that is ignored in the `.gitignore`.
 
-- [ ] Git clone this repository
 - [ ] Have `cluster-admin` access to an OpenShift 4.14+ cluster
   - [Create an OpenShift 4.14+ cluster](/docs/info-create-openshift-cluster.md)
-- [ ] [Install OpenShift CLI](https://docs.openshift.com/container-platform/4.16/cli_reference/openshift_cli/getting-started-cli.html)
-  - Have `oc` in your `$PATH`
+- [ ] Open a `bash` terminal on your local machine
```

Comment:
Would be good to also suggest SSHing to the bastion here.

Reply:
(not as the default behavior, but as an option for those who don't/can't meet the CLI tool prereqs)